package.json

```diff
@@ -2,6 +2,9 @@
   "name": "uppy-io",
   "version": "0.0.0",
   "private": true,
+  "lint-staged": {
+    "**/*": "prettier --write --ignore-unknown"
```

Comment:
we want to let remark format our markdown and mdx instead of prettier

Reply:
Why not both?